app/Http/Controllers/Auth/LoginController.php

```diff
@@ -28,6 +30,22 @@ class LoginController extends Controller
      */
     protected $redirectTo = RouteServiceProvider::HOME;
 
```

Comment:
Where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=0 Email Verification ?

Reply:
i fixed in summary doccument. 
https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=0